freemocap/tests/test_mediapipe_2d_data_shape.py

```diff
@@ -43,6 +43,6 @@ def test_mediapipe_2d_data_shape(
 
     # TODO - check number of tracked points vs 'expected' number of tracked points
 
-    assert mediapipe_2d_data.shape[3] == 2
+    assert mediapipe_2d_data.shape[3] == 3  # the shape is now 3, x y z
```

Comment:
it doesn't make sense to so "it's now 3" when you are changing this value! Don't write comments for yourself *now* write them for hypothetical future people (or really, don't write sloppy comments at all lol)